rdt/transformers/__init__.py

```diff
@@ -88,15 +90,15 @@ def get_transformer_name(transformer):
     for transformer in BaseTransformer.get_subclasses()
 }
 
-
-DEFAULT_TRANSFORMERS = {
-    'numerical': FloatFormatter(),
-    'categorical': UniformEncoder(),
-    'boolean': UniformEncoder(),
-    'datetime': UnixTimestampEncoder(),
-    'text': RegexGenerator(),
-    'pii': AnonymizedFaker(),
-}
+DEFAULT_TRANSFORMERS = WarnDict(
```

Comment:
Does this cause the warning to be raised more than once in one workflow? The cases I'm wondering are
1. If the user has multiple columns set to an sdtype of text
2. If the overall process for the HyperTransformer accesses this dict more than once
We probably want to make sure the user doesn't get spammed with this warning